docs/imaging/how-to-add-sessions.rst

```diff
@@ -5,30 +5,35 @@ Imaging:  Web GUI
 ===========================
 
 1. Copy files to a file server
-2. Open the imaging `Website`_ and log in (ask Simon Ball or Horst Obenhaus for log in credentials if this is your first log in)
+2. Open the [Web GIU website](https://datajoint.kavli.org.ntnu.no/) and log in (ask Simon Ball or Horst Obenhaus for log in credentials if this is your first log in)
```

Comment:
`[text_here](link_here)` is Markdown syntax. I have subsequently rewritten docs in ReStructuredText format (.rst), since MarkDown doesn't support a key feature - linking between distant documents in a reliable and scalable way.

The broken URL can instead be fixed in line #129, which should (I think!) update to 
```
.. _Website: https://datajoint.kavli.org.ntnu.no/